src/azure-cli/azure/cli/command_modules/acs/tests/latest/custom_preparers.py

```diff
@@ -0,0 +1,64 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from datetime import datetime
+
+from azure.cli.testsdk import CliTestError
+from azure.cli.testsdk.preparers import NoTrafficRecordingPreparer
+from azure.cli.testsdk.reverse_dependency import get_dummy_cli
+from azure_devtools.scenario_tests import SingleValueReplacer
+
+
+class AKSCustomResourceGroupPreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
```

Comment:
I would suggest to inherit from ResourceGroupPreparer.

Reply:
Sure, changed to inherit in commit [e802a6b](https://github.com/Azure/azure-cli/pull/18068/commits/e802a6b260c8f179e54a2892dee09f0a90f86c92).